### DISCUSSION TITLE
logger target

### DISCUSSION THREAD

**Discussion by macMikey** [trevordevore/levure#164] [Ideas]
I just added a logger target *answer* to a levure project, because sometimes (especially on mobile), sometimes that's your best option.
Before I issue a PR, any objections?

**Comment by trevordevore** [comment]
Honestly, I'm not sure how I feel about that. I want to know more about the problem you are trying to solve. For instance, are you adding logger statements to your app that will remain their permanently and you have users turning logging on to troubleshoot? Or are you just trying to debug some code temporarily and the logging code is only there for the troubleshooting session?

**Comment by macMikey** [comment]
I'm thinking for cases where we're remote debugging a mobe (we don't have physical access to the device) we can flip the mode to answer and then receive screenshots.

**Reply by trevordevore** [reply]
I guess that makes sense. I think in an ideal world we would have an "endpoint" target that would send the log off to a server for processing. That sounds like a lot more work though :-)

Go ahead and submit it.

**Comment by macMikey** [comment]
I thought about that, but what if there is an issue with networking in the
app that we are trying to diagnose? One of our apps that is out in the
field is exhibiting net issues on only two devices. It would be great to be
able to be able to log to endpoint, but until we determine what the network
issue is in the app, that is probably futile.
